src/main/kotlin/com/personio/synthetics/builder/api/StepBuilder.kt

```diff
@@ -87,4 +93,11 @@ class StepBuilder(
             parsingOptions.add(options)
         }
     }
+
+    /**
+     * Process the retry configuration by configuring and applying a SyntheticsTestOptionsRetry object
```

Comment:
Adjust to the laconic phrase like "Sets the retry conditions for the synthetic test"